### DISCUSSION TITLE
Can't convert book to PDF (No SN)

### DISCUSSION THREAD

**Discussion by HelterSkelter4** [noDRM/DeDRM_tools#461] [Q&A]
Hello,

I have searched for a few days on the best method to convert my book to PDF so I can read it offline. Every time I solve one issue the other arises, so I am asking here for help.

Firstly, I purchased a book from Amazon, but I don't have a Kindle. 

The book loads normally to Calibre, but I cannot convert it as it gives me the following error:

![image](https://github.com/noDRM/DeDRM_tools/assets/43636905/44eb92ee-bf49-47f2-85c2-63721df61d02)

I think it might be because I didn't insert any Kindle Serial Numbers into DeDRM, but as I don't have a kindle I can't insert any SN. Is there any way to do it without SN?  

Kindle version: 2.3.70673
Calibre version: 7.2
Book published: 2007
DeDRM version: 10.0.3
Gather KFX-ZIP version: 2.6.0

![image](https://github.com/noDRM/DeDRM_tools/assets/43636905/d3031e7b-b489-4148-8271-5b6101633302)

Thank you in advance.

**Comment by ElleKayEm** [comment]
Try DeDRM 10.0.9: https://github.com/noDRM/DeDRM_tools/releases/tag/v10.0.9
